### ISSUE TITLE
Support a properties-file (available during runtime) for ml-models (Stateless evaluation in container)

### ISSUE THREAD

**Issue by pinankg** [vespa-engine/vespa#9552]
Hello Vespa Team,
Can you please consider support a properties-file which is available during run-time along with the model ? So some meta-data e.g. threshold/label, etc can be associated with the model.
This is for the stateless evaluation of the models like (XGBoost, TensorFlow, Onnx, etc) which is supported in vespa.

Thank you,
Pinank

**Comment by Divahar2507** [comment]
can i take this issue

**Comment by bratseth** [comment]
You can describe a proposal here if you want.
